
<!-- saved from url=(0048)https://yiheng-zhu.github.io/Yiheng/papers5.html -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><style>
<!--
 li.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	text-align:justify;
	text-justify:inter-ideograph;
	font-size:10.5pt;
	font-family:"Times New Roman";
	margin-left:0cm; margin-right:0cm; margin-top:0cm}
-->
</style>

</head><body data-new-gr-c-s-check-loaded="14.1062.0" data-gr-ext-installed=""><div>
	<table border="0" id="table1" bgcolor="#FFFFFF">
<tbody><tr>
	<td valign="baseline" colspan="3" bgcolor="#ADD8E6">
		<p><b>
			<font face="Times" size="3">
				<a name="2023">Protein Complex Inter-chain Contact Map Prediction</a>
			</font>
		</b></p>
	</td>
</tr>
		<tr>
		<td valign="baseline" colspan="3">
		<ul>

			<!------------------- 2024 --------------------------------------------------------------------------------------------------------------------------->
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
		PIPENN-EMB: ensemble net and protein embeddings generalise protein interface prediction beyond homology. <b>BioRxiv</b>, 2024.
			[<a href="./papers/Project--NFS/2024--BioRxiv--PIPENN-EMB.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ProBID-Net: A Deep Learning Model for Protein-Protein Binding Interface Design. <b>Chemical Science</b>, 2024
			[<a href="./papers/Project--NFS/2024--Chemical--Science--ProBID-Net.pdf">PDF</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Protein language model-embedded geometric graphs power inter-protein contact prediction. <b>eLife</b>, 2024.
			[<a href="./papers/PCC/PLMGraph-Inter.pdf">PDF</a>]
			[<a href="https://github.com/ChengfeiYan/PLMGraph-Inter" target="_blank">Code</a>]
		</font>
	</p>
</li>
			<!------------------- 2023---------------------------------------------------------------------------------------------------------------------------->

			<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Deep transfer learning for inter-chain contact predictions of transmembrane protein complexes. <b>Nature Communications</b>, 2023.
			[<a href="./papers/PCC/DeepInter.pdf">PDF</a>]
			[<a href="http://huanglab.phys.hust.edu.cn/DeepTMP/" target="_blank">Code</a>]
		</font>
	</p>
</li>
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Protein–protein contact prediction by geometric triangle-aware protein language models. <b>Nature Machine Intelligence</b>, 2023.
			[<a href="./papers/PCC/DeepTMP.pdf">PDF</a>]
			[<a href="http://huanglab.phys.hust.edu.cn/DeepInter/" target="_blank">Code</a>]
		</font>
	</p>
</li>
<!------------------- 2022 --------------------------------------------------------------------------------------------------------------------------->
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Prediction of inter-chain distance maps of protein complexes with 2D attention-based deep neural networks. <b>Nature Communications</b>, 2022.
			[<a href="./papers/PCC/CDPred.pdf">PDF</a>]
			[<a href="https://github.com/BioinfoMachineLearning/CDPred" target="_blank">Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Improved prediction of protein-protein interactions using AlphaFold2. <b>Nature Communications</b>, 2023.
			[<a href="./papers/PCC/AF2Complex.pdf">PDF</a>]
			[<a href="https://github.com/FreshAirTonight/af2complex" target="_blank">Code</a>]
		</font>
	</p>
</li>
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			HDIContact: a novel predictor of residue–residue contacts on hetero-dimer interfaces via sequential information and transfer learning strategy. <b>Briefings in Bioinformatics</b>, 2022.
			[<a href="./papers/PCC/HDIContact.pdf">PDF</a>]
			[<a href="https://github.com/guofei-tju/zw-tju-HDIContact" target="_blank">Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Deep graph learning of inter-protein contacts. <b>Bioinformatics</b>, 2022.
			[<a href="./papers/PCC/GLINTER.pdf">PDF</a>]
			[<a href="https://github.com/zw2x/glinter" target="_blank">Code</a>]
		</font>
	</p>
</li>
			<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Inter-chain contact map prediction for protein complex based on graph attention network and triangular multiplication update. <b>BIBM</b>, 2022.
			[<a href="./papers/PCC/PGT.pdf">PDF</a>]
		</font>
	</p>
</li>
<!------------------- 2021--------------------------------------------------------------------------------------------------------------------------->
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Accurate prediction of inter-protein residue–residue contacts for homo-oligomeric protein complexes. <b>Briefings in bioinformatics</b>, 2021.
			[<a href="./papers/PCC/DeepHomo.pdf">PDF</a>]
			[<a href="http://huanglab.phys.hust.edu.cn/DeepHomo/" target="_blank">Code</a>]
		</font>
	</p>
</li>

			<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			DNCON2_Inter: predicting interchain contacts for homodimeric and homomultimeric protein complexes using multiple sequence alignments of monomers and deep learning. <b>Scientific reports</b>, 2021.
			[<a href="./papers/PCC/DNCON2_Inter.pdf">PDF</a>]
			[<a href="https://github.com/jianlin-cheng/DNCON2_Inter" target="_blank">Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
		DeepComplex: A Web Server of Predicting Protein Complex Structures by Deep Learning Inter-chain Contact Prediction and Distance-Based Modelling. <b>Frontiers in Molecular Biosciences</b>, 2021.
			[<a href="./papers/PCC/DeepComplex.pdf">PDF</a>]
			[<a href="https://github.com/jianlin-cheng/DeepComplex" target="_blank">Code</a>]
		</font>
	</p>
</li>
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
		DeepRank: a deep learning framework for data mining 3D protein-protein interfaces. <b>Nature Communication</b>, 2021.
			[<a href="./papers/PCC/DeepRank.pdf">PDF</a>]
			[<a href="https://github.com/DeepRank/deeprank" target="_blank">Code</a>]
		</font>
	</p>
</li>
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
		Tetramer protein complex interface residue pairs prediction with LSTM combined with graph representations. <b>BBA - Proteins and Proteomics</b>, 2020.
			[<a href="./papers/PCC/Tetramer.pdf">PDF</a>]
			[<a href="https://github.com/Sundw-818/Tetramer" target="_blank">Code</a>]
		</font>
	</p>
</li>



<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			ComplexContact: a web server for inter-protein contact prediction using deep learning. <b>Nucleic Acids Research</b>, 2018.
			[<a href="./papers/PCC/ComplexContact.pdf">PDF</a>][<a href="http://raptorx2.uchicago.edu/ComplexContact/" target="_blank">Code</a>]
		</font>
	</p>
</li>
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			CoRNeA: A Pipeline to Decrypt the Inter-Protein Interfaces from Amino Acid Sequence Information. <b>Biomolecules</b>, 2020.
			[<a href="./papers/PCC/CoRNeA.pdf">PDF</a>][<a href="https://github.com/krits04/cornea" target="_blank">Code</a>]
		</font>
	</p>
</li>

<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Sequence co-evolution gives 3D contacts and structures of protein complexes. <b>Biomolecules</b>, 2014.
			[<a href="./papers/PCC/2014--elife--EVcomplex.pdf">PDF</a>]
		</font>
	</p>
</li>

	</ul></td></tr></tbody></table>
<table border="0" id="table1" bgcolor="#FFFFFF">
<tbody><tr>
	<td valign="baseline" colspan="3" bgcolor="#ADD8E6">
		<p><b>
			<font face="Times" size="3">
				<a name="2023">Protein Contact Map Prediction</a>
			</font>
		</b></p>
	</td>
</tr>
		<tr>
		<td valign="baseline" colspan="3">
		<ul>
<li>
	<p style="line-height: 20px; margin-top: 10px" align="justify">
		<font face="Times" size="3">
			Contact-Distil: Boosting Low Homologous Protein Contact Map Prediction by Self-Supervised Distillation. <b>The Thirty-Sixth AAAI Conference on Artificial Intelligence</b>, 2022.
			[<a href="./papers/PCC/Contact-Distil.pdf">PDF</a>]
			[<a href="https://github.com/ChengfeiYan/PLMGraph-Inter" target="_blank">Code</a>]
		</font>
	</p>
</li>
	</ul></td></tr></tbody></table>

</div>

</body>
<grammarly-desktop-integration data-grammarly-shadow-root="true"><template shadowrootmode="open"><style>
      div.grammarly-desktop-integration {
        position: absolute;
        width: 1px;
        height: 1px;
        padding: 0;
        margin: -1px;
        overflow: hidden;
        clip: rect(0, 0, 0, 0);
        white-space: nowrap;
        border: 0;
        -moz-user-select: none;
        -webkit-user-select: none;
        -ms-user-select:none;
        user-select:none;
      }

      div.grammarly-desktop-integration:before {
        content: attr(data-content);
      }
    </style><div aria-label="grammarly-integration" role="group" tabindex="-1" class="grammarly-desktop-integration" data-content="{&quot;mode&quot;:&quot;full&quot;,&quot;isActive&quot;:true,&quot;isUserDisabled&quot;:false}"></div></template></grammarly-desktop-integration></html>