<html>
<head>
    <title>Servers</title>
    <style>
        table {
            TBBLE-LBYOUT: fixed;
        }

        td {
            word-wrap: break-word;
        }
    </style>
</head>
<body>
<div align="center">
    <table width="880" id="table1" bgcolor="#ffffff" border="0">
        <tbody>
        <tr>
            <td width="880" valign="baseline">
                <hr/>
            </td>
        </tr>
        <tr>
            <td width="880" valign="middle" halign="left">
						<span style="font-family:georgia;font-size:18px;">
							<a name="Publications_">Bioinformatics Servers</a>&nbsp;(<strong>18</strong> Servers), Programmed by Zi Liu</span></strong>
                </span>
                <hr/>
            </td>
        </tr>
        <!-- xxxx -->
        <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/TMsearch.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                            <p style="text-align:left;">
														<span style="font-family:georgia;font-size:12px;">
															<span style="color:#0000ff;">
																<a style="color:#0000ff;" href="https://zhanggroup.org/TM-search" target="_blank"><strong>TM-search</strong></a>
															</span>
															<br/>
															<span style="color:#0000ff;"><strong>Category</strong>: </span>[Protein structure database search]
															<br/>
															<span style="color:#0000ff;"><strong>Description</span></strong>: TM-search: An Efficient and Effective Tool for Protein Structure Database Search
															<br/>
															<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Journal of Chemical Information and Modeling</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="https://doi.org/10.1021/acs.jcim.3c01455" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1021/acs.jcim.3c01455
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2024</strong>
														</span>
                            </p>
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/ICCPred.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
<p style="text-align:left;">
														<span style="font-family:georgia;font-size:12px;">
															<span style="color:#0000ff;">
																<a style="color:#0000ff;" href="http://202.119.84.36:3079/uldna/" target="_blank"><strong>ULDNA</strong></a>
															</span>
															<br/>
															<span style="color:#0000ff;"><strong>Category</strong>: </span>[Protein-DNA binding site prediction]
															<br/>
															<span style="color:#0000ff;"><strong>Description</span></strong>: ULDNA: Integrating unsupervised multi-source language models with LSTM-attention network for high-accuracy protein-DNA binding site prediction
															<br/>
															<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Briefings in Bioinformatics</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="https://doi.org/10.1093/bioinformatics/btab667" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1093/bib/bbae040
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2024</strong>
														</span>
                            </p>      
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
        <!-- xxxx -->
        <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/ULDNA.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                                                    <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;"
                                                   href="https://github.com/yiheng-zhu/ICCPred" target="_blank"><strong>ICCPred</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Protein-protein contact map prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: Integrating unsupervised Language Model with Multi-View Multiple Sequence Alignments for High-Accuracy Inter-Chain Contact Prediction
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Computers in Biology and Medicine</strong>
											<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="https://doi.org/10.1016/j.compbiomed.2023.107529" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1016/j.compbiomed.2023.107529
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2023</strong>
										</span>
                            </p>
                            
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/Stackingac4C.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
                            <p style="text-align:left;">
														<span style="font-family:georgia;font-size:12px;">
															<span style="color:#0000ff;">
																<a style="color:#0000ff;" href="https://github.com/louliliang/ST-ac4C" target="_blank"><strong>Stacking-ac4C</strong></a>
															</span>
															<br/>
															<span style="color:#0000ff;"><strong>Category</strong>: </span>[N4-acetylcytidine site prediction]
															<br/>
															<span style="color:#0000ff;"><strong>Description</span></strong>: Stacking-ac4C: an ensemble model using mixed features for identifying n4-acetylcytidine in mRNA
															<br/>
															<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Frontiers in Immunology</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="https://doi.org/10.3389/fimmu.2023.1267755" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.3389/fimmu.2023.1267755
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2023</strong>
														</span>
                            </p>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
        <!-- xxxx -->
        <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/cpxDeepMSA.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                            <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;" href="https://zhanggroup.org/cpxDeepMSA/"
                                                   target="_blank"><strong>cpxDeepMSA</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Multiple Sequence Alignment Construction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: cpxDeepMSA: A Deep Cascade Algorithm for Constructing Multiple Sequence Alignments of Protein–Protein Interactions
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>International Journal of Molecular Sciences</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="https://doi.org/10.3390/ijms23158459" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.3390/ijms23158459
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2021</strong>
										</span>
                            </p>
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/LS-align.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
                        <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;"
                                                   href="https://zhanglab.ccmb.med.umich.edu/LS-align/" target="_blank"><strong>LS-align</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Ligand structural alignment]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: LS-align is an algorithm designed for atom-level structural comparison of ligand molecules.
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Bioinformatics</strong>
											<br/>
											<span style="color:#0000ff;"><strong><a
                                                    href="https://doi.org/10.1093/bioinformatics/bty081" target=_blank>DOI:</a></strong></span>&nbsp;10.1093/bioinformatics/bty081
											<br/>
											<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2018</strong>
										</span>
                            </p>
                           
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
        <!-- xxxx -->
        <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/iDNA-Methyl.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                            <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;" href="http://www.jci-bioinfo.cn/iDNA-Methyl"
                                                   target="_blank"><strong>iDNA-Methyl</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[DNA methylation sites Prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: iDNA-Methyl: Identifying DNA methylation sites via pseudo trinucleotide composition
											<br>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Analytical Biochemistry</strong>
											<br/>
											<span style="color:#0000ff;"><strong><a
                                                    href="https://doi.org/10.1016/j.ab.2014.12.009"
                                                    target=_blank>DOI:</a></strong></span>&nbsp;10.1016/j.ab.2014.12.009
											<br/>
											<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2015</strong>
										</span>
                            </p>
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/pRNAm-PC.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
                           <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;"
                                                   href="http://www.jci-bioinfo.cn/pRNAm-PC" target="_blank"><strong>pRNAm-PC</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[RNA N6-methyldenosine site prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: pRNAm-PC: Predicting N6-methyladenosine sites in RNA sequences via physicalechemical properties
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Analytical Biochemistry</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="http://dx.doi.org/10.1016/j.ab.2015.12.017" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1016/j.ab.2015.12.017
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2016</strong>
										</span>
                            </p>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
        <!-- xxxx -->
            <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/pSuc-Lys.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                            <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;" href="http://www.jci-bioinfo.cn/pSuc-Lys"
                                                   target="_blank"><strong>pSuc-Lys</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Lysine succinylation sites prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: pSuc-Lys: Predict lysine succinylation sites in proteins with PseAAC and ensemble random forest approach
											<br>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Journal of Theoretical Biology</strong>
											<br/>
											<span style="color:#0000ff;"><strong><a
                                                    href="https://doi.org/10.1016/j.jtbi.2016.01.020"
                                                    target=_blank>DOI:</a></strong></span>&nbsp;10.1016/j.jtbi.2016.01.020
											<br/>
											<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2016</strong>
										</span>
                            </p>
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/iPPI-Esml.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
                           <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;"
                                                   href="http://www.jci-bioinfo.cn/iPPI-Esml" target="_blank"><strong>iPPI-Esml</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Protein-protein interaction prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>:iPPI-Esml: An ensemble classiﬁer for identifying the interactions of proteins by incorporating their physicochemical properties and wavelet transforms into PseAAC
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Analytical Biochemistry</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="https://doi.org/10.1016/j.jtbi.2015.04.011" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1016/j.jtbi.2015.04.011
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2015</strong>
										</span>
                            </p>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
         <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/iDrug-Target.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                            <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;" href="http://www.jci-bioinfo.cn/iDrug-Target"
                                                   target="_blank"><strong>iDrug-Target</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Drug and protein prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: iDrug-Target: predicting the interactions between drug compounds and target proteins in cellular networking via benchmark dataset optimization approach
											<br>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Journal of Biomolecular Structure and Dynamics</strong>
											<br/>
											<span style="color:#0000ff;"><strong><a
                                                    href="http://dx.doi.org/10.1080/07391102.2014.998710"
                                                    target=_blank>DOI:</a></strong></span>&nbsp;10.1080/07391102.2014.99871
											<br/>
											<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2015</strong>
										</span>
                            </p>
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/iPPBS-PseAAC.jpg" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
                           <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;"
                                                   href="http://www.jci-bioinfo.cn/iPPI-Esml" target="_blank"><strong>iPPBS-PseAAC</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Protein-protein binding sites prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>:Identification of protein-protein binding sites by incorporating the physicochemical properties and stationary wavelet transforms into pseudo amino acid composition
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Journal of Biomolecular Structure & Dynamics</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="https://doi.org/10.1080/07391102.2015.1095116" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1080/07391102.2015.1095116
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2015</strong>
										</span>
                            </p>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
        <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/iCataly-PseAAC.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                            <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;" href="http://www.jci-bioinfo.cn/iCataly-PseAAC"
                                                   target="_blank"><strong>iCataly-PseAAC</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Enzymes catalytic sites prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: iCataly-PseAAC: Identification of Enzymes Catalytic Sites Using Sequence Evolution Information with Grey Model GM (2,1)
											<br>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>The Journal of Membrane Biology</strong>
											<br/>
											<span style="color:#0000ff;"><strong><a
                                                    href="http://dx.doi.org/10.1007/s00232-015-9815-8"
                                                    target=_blank>DOI:</a></strong></span>&nbsp;10.1007/s00232-015-9815-8
											<br/>
											<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2015</strong>
										</span>
                            </p>
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/iSuc-PseOpt.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
                           <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;"
                                                   href="http://www.jci-bioinfo.cn/iSuc-PseOpt" target="_blank"><strong>iSuc-PseOpt</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Lysine succinylation sites prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>:iSuc-PseOpt: Identifying lysine succinylation sites in proteins by incorporating sequence-coupling effects into pseudo components and optimizing imbalanced training dataset
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Analytical Biochemistry</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="http://dx.doi.org/10.1016/j.ab.2015.12.009" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1016/j.ab.2015.12.009
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2015</strong>
										</span>
                            </p>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
                <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/M5C-HPCRC.jpg" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                            <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;" href="http://cslab.just.edu.cn:8080/M5C-HPCR/"
                                                   target="_blank"><strong>M5C-HPCRC</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[5-methylcytosine site prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: Accurate RNA 5-methylcytosine site prediction based on heuristic physical-chemical properties reduction and classifier ensemble
											<br>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Analytical Biochemistry</strong>
											<br/>
											<span style="color:#0000ff;"><strong><a
                                                    href="https://doi.org/10.1016/j.ab.2018.03.027"
                                                    target=_blank>DOI:</a></strong></span>&nbsp;10.1016/j.ab.2018.03.027
											<br/>
											<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2018</strong>
										</span>
                            </p>
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/iAFP-Ense.jpg" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
                           <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;"
                                                   href="http://www.jci-bioinfo.cn/iAFP-Ense" target="_blank"><strong>iAFP-Ense</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Protein antifreeze prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>:iAFP-Ense: An Ensemble Classifier for Identifying Antifreeze Protein by Incorporating Grey Model and PSSM into PseAAC
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>The Journal of Membrane Biology</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="https://doi.org/10.1007/s00232-016-9935-9" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1007/s00232-016-9935-9
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2016</strong>
										</span>
                            </p>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
                <tr>
            <td width="880">
                <table width="880" border="0">
                    <tbody>
                    <tr>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/pSumo-CD.jpg" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" align="center">
                            <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;" href="http://www.jci-bioinfo.cn/pSumo-CD"
                                                   target="_blank"><strong>pSumo-CD</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[Sumoylation sites sites prediction]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>: pSumo-CD: predicting sumoylation sites in proteins with covariance discriminant algorithm by incorporating sequence-coupled effects into general PseAAC
											<br>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>Bioinformatics</strong>
											<br/>
											<span style="color:#0000ff;"><strong><a
                                                    href="https://doi.org/10.1093/bioinformatics/btw387"
                                                    target=_blank>DOI:</a></strong></span>&nbsp;10.1093/bioinformatics/btw387
											<br/>
											<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2015</strong>
										</span>
                            </p>
                        </td>
                        <td>
                            <img width="137" height="157"
                                 style="border:1px solid #0000ff;border-image:none;width:137px;height:157px;padding-top:1px;padding-bottom:1px;"
                                 src="./images/TargetM6A.png" border="0" vspace="0" hspace="0"/>
                        </td>
                        <td width="285" style="text-align:left;">
                           <p style="text-align:left;">
										<span style="font-family:georgia;font-size:12px;">
											<span style="color:#0000ff;">
												<a style="color:#0000ff;"
                                                   href="https://csbio.njust.edu.cn/bioinf/TargetM6A/" target="_blank"><strong>TargetM6A</strong></a>
											</span>
											<br/>
											<span style="color:#0000ff;"><strong>Category</strong>: </span>[N6-methyladenosine sites]
											<br/>
											<span style="color:#0000ff;"><strong>Description</span></strong>:TargetM6A: Identifying N6-Methyladenosine Sites From RNA Sequences via Position-Specific Nucleotide Propensities and a Support Vector Machine
											<br/>
											<span style="color:#0000ff;"><strong>Journal</strong></span>: <strong>IEEE TNB</strong>
															<br/>
															<span style="color:#0000ff;">
																<strong>
																	<a href="http://dx.doi.org/10.1016/j.ab.2015.12.009" target=_blank>DOI:</a>
																</strong>
															</span>&nbsp;10.1016/j.ab.2015.12.009
															<br/>
															<span style="color:#0000ff;"><strong>Year</strong></span>: <strong>2015</strong>
										</span>
                            </p>
                        </td>
                    </tr>
                    </tbody>
                </table>
            </td>
        </tr>
               
        </tbody>
    </table>
</div>
</body>
</html>
